packages/create-vuestic/src/steps/2.addVuestic/vue-app.ts

```diff
@@ -41,7 +41,7 @@ export const addVuesticToVue3App = async () => {
   let htmlSource = await readFile(htmlPath, 'utf-8')
   htmlSource = insertHead(htmlSource, [
     `<link href="https://fonts.googleapis.com/css2?family=Source+Code+Pro&family=Source+Sans+Pro:wght@400;700&display=swap" rel="stylesheet">`,
-    `<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">`
+    `<link href="https://fonts.googleapis.com/icon?family=Material+Symbols+Outlined" rel="stylesheet">`
```

Comment:
We need to update IconsConfig to use material-symbols but default, instead of material icons.
I expect updating nuxt module in my project and all icons still work, even if the're symbols now.

Right now, creating new vuestic project will install Material Symbols, but we use material-icons in IconsConfig...

Reply:
We should release both projects simultaneously then.